### ISSUE TITLE
v4 - Split button dropdowns get excess width when active on mobile devices

### ISSUE THREAD

**Issue by GeoffYoung** [twbs/bootstrap#17321]
In the split button dropdowns when the dropdown menu is opened the `dropdown-toggle` gets some extra width applied.

I'm seeing this on Chrome 44 for android and Safari 8 on iOS

![photo aug 25 8 52 35 pm](https://cloud.githubusercontent.com/assets/869227/9483348/1bff20ac-4b6d-11e5-9394-d0f176444b9d.png)

**Comment by GeoffYoung** [comment]
So there's padding of `.375rem 1rem` applied to `.btn` which is causing 16px padding on the left and right.

It's normally being overridden by
```css
.btn-group>.btn+.dropdown-toggle {
    padding-right: 8px;
    padding-left: 8px;
}
```
but when the dropdown opens `<div class="dropdown-backdrop"></div>` is created in between that `.btn` and the `.dropdown-toggle`, which seems to prevent the sibling selector from applying.

**Comment by patrickhlauke** [comment]
confirming this for other platforms as well (Android/Chrome, Windows 10 Mobile/Microsoft Edge)

![split-android-chrome](https://cloud.githubusercontent.com/assets/895831/11978705/8b970436-a985-11e5-980f-0f209145e002.png)

![split-win10](https://cloud.githubusercontent.com/assets/895831/11978761/e5738b5a-a985-11e5-8e9b-ef78ebc4b84c.png)

**Comment by mdo** [comment]
This was addressed with #18807 in Alpha 3. Thanks!
